conans/client/cache/cache.py

```diff
@@ -155,6 +157,22 @@ def config(self):
             self._config = ConanClientConfigParser(self.conan_conf_path)
         return self._config
 
+    @property
+    def new_config_path(self):
+        return os.path.join(self.cache_folder, "conan_conf.txt")
+
+    @property
+    def new_config(self):
+        """ this is the new conan_conf.txt to replace the old conan.conf that contains
```

Comment:
I would prefer .conf extension. .txt is generic, conf can be interpreted as .ini, even for code editors it can used for highlight.

Reply:
That is a temporary name, it might be possible that everything become python files.
`conan_conf.conf` would be ugly, `conan_conf.ini` in the meantime?

Reply:
yes, temporary name, to be changed.

Reply:
I'd also prefer a different extension to indicate it's a config file, not a generic text file. something like `.cfg`, `.conf` or `.ini` will work.